webapp/src/pages/ChatsPage/template/ChatsPageTemplate.tsx

```diff
@@ -35,18 +34,18 @@ export default function ChatsPageTemplate({
   const navigate = useNavigate();
   const overridePermissions =
     buttonUrl.slice(0, ADMIN_URL.length) === ADMIN_URL;
-  const mapChatToRow = (chatroom: Chatroom): RowItem => {
+  const mapChatToRow = (chatroom: GenericChat): RowItem => {
     return {
       iconVariant: IconVariant.ARROW_FORWARD,
       avatarProps: {
         url: `${AVATAR_EP_URL}/${chatroom.avatarId}`,
+        XCoordinate: chatroom.avatarX,
+        YCoordinate: chatroom.avatarY,
```

Comment:
maybe we would like to report here the online/offline status of users in case of one to one chat. But maybe that's something for a follow up ticket.

Reply:
done